.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+<!-- If this PR closes a JIRA ticket, make sure the title starts with the JIRA issue number,
+for example RCAL-1234: <Fix a bug> -->
+Resolves [RCAL-nnnn](https://jira.stsci.edu/browse/RCAL-nnnn)
+
+<!-- If this PR closes a GitHub issue, reference it here by its number -->
+Closes #
+
+<!-- describe the changes comprising this PR here -->
+This PR addresses ...
+
+**Checklist**
+- [ ] Added entry in `CHANGES.rst` under the corresponding subsection
+- [ ] Passed romancal regression testing on Jenkins / PLWishMaster. Link: https://plwishmaster.stsci.edu:8081/job/RT/job/romancal/XXX/
```

Comment:
Should tests and documentation be considered when submitting PR?

Reply:
I didn't think they were needed as RDM is mostly a test package, in a sense, and has little online documentation, but I am happy to add them if you wish.

Reply:
Added.